<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorldClockWidget</class>
 <widget class="QWidget" name="WorldClockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="city0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>City 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="city1">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>30</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>City 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="city2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>City 3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="city3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>City 4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="city4">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>170</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>City 5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="city5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>170</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>City 6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="val1">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>70</y>
     <width>230</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeMono</family>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: black;
border-radius: 6px;</string>
   </property>
   <property name="text">
    <string>12:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="val2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>230</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeMono</family>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: black;
border-radius: 6px;</string>
   </property>
   <property name="text">
    <string>12:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="val4">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>210</y>
     <width>230</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeMono</family>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: black;
border-radius: 6px;</string>
   </property>
   <property name="text">
    <string>12:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="val5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>210</y>
     <width>230</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeMono</family>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: black;
border-radius: 6px;</string>
   </property>
   <property name="text">
    <string>12:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="val3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>230</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeMono</family>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: black;
border-radius: 6px;</string>
   </property>
   <property name="text">
    <string>12:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="val0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>230</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeMono</family>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: black;
border-radius: 6px;</string>
   </property>
   <property name="text">
    <string>12:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
